test/regress/regress0/arrays/issue8103-1-weak-equiv-models.smt2

```diff
@@ -0,0 +1,11 @@
+; COMMAND-LINE: --check-models --arrays-weak-equiv
```

Comment:
Is this configuration necessary?  I'd prefer keeping `check-models` off command line in regressions.

Reply:
As opposed to just running with `--produce-models`? I'd be fine with that.

Reply:
Yes, you already have `--produce-models --arrays-weak-equiv` in the line below.

Reply:
Yeah, the idea was to make sure that we catch it in both cases. It's not super important, though. I've removed it.